metadata/site_import/meta/system-objecttype-extensions.xml

```diff
@@ -120,6 +120,14 @@
                 <min-length>0</min-length>
                 <field-length>0</field-length>
             </attribute-definition>
+            <attribute-definition attribute-id="adyenMainPaymentInstrument">
+                <display-name xml:lang="x-default">adyenMainPaymentInstrument</display-name>
+                <description xml:lang="x-default">This setting is used to determine which payment method is used during checkout</description>
+                <type>text</type>
```

Comment:
Since we're only accepting 2 values, maybe we can set the type to enum of strings and define those 2 values?

Reply:
Added this as an enum. Great suggestion 👍